### DISCUSSION TITLE
Automatic backlinks?

### DISCUSSION THREAD

**Discussion by Guillawme** [glushchenko/fsnotes#1662] [Ideas]
Hello,

I am trying FSNotes these days and I like it overall. There is one thing I am missing, but maybe it is there and I simply haven't found it yet: can it do automatic backlinks? also known as bi-directional links.

Say I am in "Note A" which contains an internal link `[[some topic]]` linking to "Note B". I would like for "Note B" to automatically contain a link back to "Note A". Can FSNotes do this? If so, how? (it doesn't seem to happen with the default configuration).

Thanks!

**Comment by tlrand** [comment]
I am on the fence on this one. I wanted them, but found the View/Backlinks really easy to use, it doesn't clutter the parent doc, and seems like the very FSNotesy way to do things.
